scripts/pylint_extensions.py

```diff
@@ -757,6 +757,33 @@ def process_module(self, node):
                         'backslash-continuation', line=line_num + 1)
 
 
+class FunctionArgsOrderChecker(checkers.BaseChecker):
+    """Checker for order of arguments. It checks
+       that class arguments comes first.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+    name = 'function-args-order'
+    priority = -1
+    msgs = {
+        'C0005': ('Wrong order of arguments '
+                  'self should come first',
+                  'wrong-ord-args-self',
+                  'self should come first',),
+        'C0006': ('Wrong order of arguments '
```

Comment:
Break after `(` and change this to `Wrong order of arguments in function definition. 'cls' should come first.`